pkg/commands/system.go

```diff
@@ -200,3 +218,69 @@ func GetOAMReleaseVersion() (string, error) {
 	}
 	return "", errors.New("oam-kubernetes-runtime not found in your kubernetes cluster, try `vela install` to install")
 }
+
+func PrintTrackVelaRuntimeStatus(ctx context.Context, c client.Client, ioStreams cmdutil.IOStreams) error {
+	trackTimeout := 5 * time.Minute
+	trackInterval := 2 * time.Second
+
+	ioStreams.Info("\nIt may take 1-2 minutes before KubeVela runtime is ready.")
+	start := time.Now()
+	spiner := newTrackingSpinner("Waiting KubeVela runtime ready to serve ...")
+	spiner.Start()
+	defer spiner.Stop()
+
+TrackRuntimeStatusLoop:
+	time.Sleep(trackInterval)
+	timeConsumed := int(time.Since(start).Seconds())
+	applySpinnerNewSuffix(spiner, fmt.Sprintf("Waiting KubeVela runtime ready to serve (timeout %d/%d seconds) ...",
+		timeConsumed, int(trackTimeout.Seconds())))
+
+	sts, podName, err := getVelaRuntimeStatus(ctx, c)
+	if err != nil {
+		return err
+	}
+	switch sts {
+	case NotInstalled:
+		ioStreams.Info(fmt.Sprintf("\n%s %s", emojiFail, "KubeVela runtime is not installed."))
```

Comment:
This should be timeout instead of not installed.

Reply:
If there's some error that lead to not installed, could we get the error detail?